meta-ostro-xt/recipes-swupd/swupd-client/swupd-client_%.bbappend

```diff
@@ -3,6 +3,8 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI_append = " \
     file://ostroprojectorg.key \
 "
+SWUPD_VERSION_URL ?= "https://download.ostroproject.org/updates/ostro-os-xt/milestone/intel-corei7-64/ostro-xt-image-swupd"
```

Comment:
@rojkov We don't plan to update ostro-os so can you please add the changes in http://git.yoctoproject.org/cgit/cgit.cgi/meta-swupd/commit/?h=pre-sstate&id=606ce9e8e266465619e552489425c7e1a15bec39 here too?